app/routes/planet_routes.py

```diff
@@ -1,10 +1,40 @@
-from flask import Blueprint, make_response, abort
-from ..models.planet import planets
+from flask import Blueprint, make_response, abort, request, Response
+from app.models.planet import Planet
+from ..db import db
 
 planets_bp = Blueprint("planets_bp",__name__,url_prefix="/planets")
 
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    description = request_body["description"]
+    galaxy = request_body["galaxy"]
+
+    new_planet = Planet(name=name, description=description, galaxy=galaxy)
+    db.session.add(new_planet)
+    db.session.commit()
+
+    response = new_planet.to_dict()
+
+    return response, 201
+
+
 @planets_bp.get("")
 def get_all_planets():
+
+    query = db.select(Planet)
+    description_param = request.args.get("description")
+    if description_param:
+        query = query.where(Planet.description.ilike(f"%{description_param}%"))
+
+    galaxy_param = request.args.get("galaxy")
+    if galaxy_param:
+        query = query.where(Planet.galaxy.ilike(f"%{galaxy_param}%"))
+
+    #query = query.order_by(Planet.id)
+    planets = db.session.scalars(query.order_by(Planet.id))
```

Comment:
Since we decomposed the query into separate steps, I'd probably lean towards the following for consistency
```py
    query = query.order_by(Planet.id)
    planets = db.session.scalars(query)
```